packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import KeyCode from 'keycode-js';
+import DropdownButtonBase from './_DropdownButtonBase';
+import styles from './DropdownButton.module.scss';
+import Button from './Button';
+import SplitButton, { Variants as SplitButtonVariants } from './SplitButton';
+
+const cx = classNames.bind(styles);
+
+const Variants = {
+  NEUTRAL: 'neutral',
+  EMPHASIS: 'emphasis',
+  GHOST: 'ghost',
+};
+
+const propTypes = {
+  /**
+   * The options to display in the dropdown. Should be comprised of the subcomponent `Button`.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Determines whether the dropdown should be disabled.
+   */
+  disabled: PropTypes.bool,
```

Comment:
This should be `isDisabled` to match the other boolean props throughout. You'll just have to map isDisabled->disabled for the button element.

Reply:
Funny you mention that, it was originally like that. Stephen suggested to have it changed the other way https://github.com/cerner/terra-core/issues/2216#issuecomment-502746195

Reply:
@StephenEsser
> I think isDisabled should be disabled. For html attributes we generally match naming conventions unless there is a chance they can conflict. But in this context they share the same meaning.

Terra's standalone Button component uses `isDisabled`: https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L50

But the various Form components use `disabled`: https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/src/Textarea.jsx#L39

I personally find the difference between the two naming schemes jarring, but I'm really just looking for some consistency in this pattern. Are we expecting to update <Button /> to remove `isDisabled` at some point?

Reply:
I'll try to find the previous thread, it might be on the locked war-room notes. We had discussed in early terra development that we would match boolean properties that map directly to the html property. 

For example:
- disabled
- required
- readonly

cc: @bjankord - Do you remember this discussion / have opinions on this?

Reply:
![](https://media.giphy.com/media/nWg4h2IK6jYRO/giphy.gif)

Reply:
I have returned from searching through the email graveyard. 

First I stumbled upon this:
https://github.com/cerner/terra-core/pull/305#discussion_r115284789

Which lead me to this: 

https://engineering.cerner.com/terra-ui/#/contributing/terra-ui/conventions

Which contains this:

![image](https://user-images.githubusercontent.com/6720991/60911045-ea5f4280-a247-11e9-89c6-c523c817c921.png)

Reply:
Stephen bringing the receipts. I guess the sacred texts have spoken. 

(It's still weird)